### DISCUSSION TITLE
How to fund local LNbits wallet

### DISCUSSION THREAD

**Discussion by n-e-t-d-i-v-e-r** [lnbits/lnbits#1872] [Q&A] [answered]
Hi there
I’m running a LNbits instance on my local raspibolt environment and it’s up an running so far. But now I’m asking myself, how to fund the local LNbits-wallet. Is there anybody who can describe me the process or a step by step manual?

Is it only possible to fund local LNbits with the local LND or coreLightning-wallet? and if yes, how is this working with RTL and/or thunderhub?

**Comment by arbadacarbaYK** [comment]
Hey, info on this and a lot of other topics can be found in the LNbits wiki.
You can add funding sources in your SuperUser-GUI under Manage Server
https://github.com/lnbits/lnbits/wiki/LNbits-Documentation#funding-sources

To top up from the set funding source press the button TopUp 
<img width="838" alt="Bildschirmfoto 2023-08-09 um 21 40 51" src="https://github.com/lnbits/lnbits/assets/63317640/dfd4b46f-26dc-42e0-a9c5-1ab83ffdf45d">

<img width="611" alt="Bildschirmfoto 2023-08-09 um 21 41 00" src="https://github.com/lnbits/lnbits/assets/63317640/645f084b-7a84-4d0f-8485-a7c52ed65db1">

Have fun :D

**Comment by n-e-t-d-i-v-e-r** [comment, marked as answer]
this documentation brought me to succeed:
https://github.com/lnbits/lnbits/wiki/LNbits-Extensions
https://github.com/lnbits/lnbits/wiki/LNbits-Documentation#funding-sources

thank you for supporting me.
